components/Common/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Banner from './';
+
+const meta: Meta<typeof Banner> = {
```

Comment:
Since this meta is used only on the default export, can't you make it an inline export? (I doubt you need to make a `typeof Banner` here).